docs/USER_GUIDE.md

```diff
@@ -45,20 +45,17 @@ Endpoints]:
 
 #### Creating a New VPC
 
-If you do not have an existing VPC you wish to use, a VPC template is available
-as part of this solution which can be deployed separately to the main stack.
-This template will create a VPC with private subnets and all the relevant VPC
-Endpoints required by the Amazon S3 Find and Forget solution. To deploy this
-template, use the VPC Template "Deploy to AWS button" in
-[Deploying the Solution](#deploying-the-solution) then follow steps 5-9. The
-**Outputs** tab will contain the subnets and security group IDs to use as inputs
-for the main stack.
+If you do not have an existing VPC you wish to use, a solution provided VPC can
+be created by setting _DeployVpc_ to true when deploying the solution. The
+solution provided VPC only has private subnets and has all the relevant VPC
+Endpoints required by the Amazon S3 Find and Forget solution.
```

Comment:
```suggestion
By default the CloudFormation template will create a new VPC that has been
purpose-built for the solution. The VPC includes VPC endpoints for the
aforementioned services, and does not provision internet connectivity.

You can use the provided VPC to operate the solution with no further
customisations. However, if you have more complex requirements it is
recommended to use an existing VPC as described in the following section.
```

I think it makes sense to position the provided VPC as the new default -- that unless you want/need to BYO, just use it and you're set.